### DISCUSSION TITLE
Minimum number of species

### DISCUSSION THREAD

**Discussion by juliayork** [hahnlab/CAGEE#18] [Q&A] [answered]
Hello, I just read the CAGEE paper and I am considering using it for my data. I understood from the paper that there is no minimum number of species, is that correct? For example, I have three species, would that be ok? In that case, I could define up to 4 parameters across the tree, as long as they are continuous, correct? Thanks for your help in my understanding.

**Comment by benfulton** [comment, marked as answer]
Yes, even two species are okay to use!

(though one wouldn’t be able to say much)

**Comment by juliayork** [comment]
Okay, thank you
